corehq/apps/users/models.py

```diff
@@ -771,6 +771,19 @@ def __eq__(self, other):
         return all(getattr(self, p) == getattr(other, p) for p in self.properties())
 
 
+class LastSubmission(DocumentSchema):
+    submission_date = DateTimeProperty()
+    app_id = StringProperty()
+    build_id = StringProperty()
+    build_version = IntegerProperty()
+    commcare_version = StringProperty()
+
+
+class ReportingMetadata(DocumentSchema):
```

Comment:
this might be over-engineering, we could possible want to just stick `last_submission` on the user model itself